nucleus/common/common-util/src/test/java/com/sun/enterprise/universal/process/ProcessManagerTest.java

```diff
@@ -73,4 +76,76 @@ private List<String> hugeInput() {
 
         return l;
     }
+
+    @Test
+    void testCommandExecution() {
+        ProcessManager pm = new ProcessManager("sh", "-c", "echo hello; sleep 1");
+        try {
+            int exitCode = pm.execute();
+            assertEquals(0, exitCode);
+            assertEquals("hello\n", pm.getStdout());
+        } catch (ProcessManagerException e) {
+            fail("Process execution failed");
```

Comment:
I think, it's better to use new JUnit Jupiter API, i.e. `assertAll`, `assertDoesNotThrow` and `assertThrows` instead of `fail`. E.g.,

```java
...
int exitCode = assertDoesNotThrow(pm::execute);
assertAll(
    () -> assertEquals(0, exitCode),
    () -> assertEquals("hello\n", pm.getStdout()
);
...
```

In my opinion this make a code more clean.